frontend/storybook/src/components/entry/homepage/sections/recommends-paragraph.ts

```diff
@@ -0,0 +1,64 @@
+import "@elements/entry/home/TOSORT/icon-wparagraph";
+import "@elements/entry/home/sections/recommend";
+ 
+export default {
+  title: 'Homepage',
+}
+
+
+const STR_PATH_HEBREW="Image_Hebrew.png";
+const STR_TITLE_HEBREW = "Hebrew";
+const STR_PARAGRAPH_HEBREW = "1,850 JIGs";
+ const STR_DARKGRAY = "#272727";
```

Comment:
icon paths and color properties should not be in STR_ prefixes. The intent of the STR_ prefixes is that we will eventually move them out of code completely and they will be set in a database for copywriters/translators.

For story mock data, just set the default properties in DEFAULT_ARGS directly (and other radio options in `argTypes`)

Reply:
Changed, apart from color. This will be fixed when Gali has added the hex colors with name